packages/css-data/src/schema.ts

```diff
@@ -97,6 +97,15 @@ const ArrayValue = z.object({
 });
 export type ArrayValue = z.infer<typeof ArrayValue>;
 
+export const TupleValueItem = z.union([UnitValue, KeywordValue, UnparsedValue]);
+export type TupleValueItem = z.infer<typeof TupleValueItem>;
+
+export const TupleValue = z.object({
+  type: z.literal("tuple"),
+  value: z.array(TupleValueItem),
+});
+export type TupleValue = z.infer<typeof TupleValue>;
+
 export const validStaticValueTypes = [
```

Comment:
We need to document what means `validStaticValueTypes` every time I see this I'm using the search

Reply:
commented